app/views/sessions/new.html.haml

```diff
@@ -1,7 +1,7 @@
-- title "Sign in"
+- title "Sign in to Coderwall"
 
 .container
-  %h2
+  %h1
```

Comment:
Is this for visual design or for SEO? If SEO, can you add a comment and make sure it's not affecting the visual (or should it?). Post a screenshot of before/after to help us understand.

Reply:
This is for SEO. Check the last screenshot why h1 tag should be there.
It has some visual affect, i.e. text became bigger.
![Screenshot from 2019-06-05 13-33-47](https://user-images.githubusercontent.com/5982702/58950383-5822d500-8797-11e9-8023-f2f52c7fac40.png)
![Screenshot from 2019-06-05 13-34-19](https://user-images.githubusercontent.com/5982702/58950384-58bb6b80-8797-11e9-96e8-e68dec46b05e.png)
![Screenshot from 2019-06-05 13-39-09](https://user-images.githubusercontent.com/5982702/58950385-58bb6b80-8797-11e9-933f-1a0510782f6c.png)